src/pages/WalletsMigration/WalletsMigration.js

```diff
@@ -0,0 +1,127 @@
+// @flow strict
+
+import Promise from 'bluebird'
+import React, { Component } from 'react'
+import { compose } from 'redux'
+import { connect } from 'react-redux'
+import { withI18n } from '@lingui/react'
+import { type I18n } from '@lingui/core'
+
+import {
+  Form,
+  type FormRenderProps,
+} from 'react-final-form'
+
+import { migrate } from 'store/migrations'
+import { gaSendEvent } from 'utils/analytics'
+import { WalletPasswordForm } from 'components'
+
+import styles from './walletsMigration.m.scss'
+
+type Props = {|
+  +i18n: I18n,
+  +hint: string,
+|}
+
+const INITIAL_VALUES: FormFields = {
+  password: '',
+}
+
+class WalletsMigration extends Component<Props> {
+  handleSubmit = async (values: FormFields): Promise<?FormFields> => {
+    const { i18n }: Props = this.props
+
+    gaSendEvent(
+      'MigrateWallets',
+      'PasswordEntered',
+    )
+
+    try {
+      await migrate(values.password || '')
+
+      gaSendEvent(
+        'MigrateWallets',
+        'WalletsMigrationSuccess',
+      )
+
+      window.location.reload()
+
+      return null
+    } catch (error) {
+      console.error(error)
```

Comment:
Let's store some important exceptions to Google Analytics, and in one of the next releases migrate it to Sentry.

GA exceptions docs: https://developers.google.com/analytics/devguides/collection/analyticsjs/exceptions